src/Dependencies/Composition/project.json

```diff
@@ -1,9 +1,12 @@
 {
   "supports": {},
   "dependencies": {
-    "Microsoft.Composition": "1.0.27"
+    "Microsoft.Composition": "1.0.27",
+    "System.Runtime": "4.1.0"
```

Comment:
This should flow from a reference to `NETStandard.Library` and `Microsoft.NETCore.Portable.Compatibility`